linera-rpc/src/grpc/client.rs

```diff
@@ -330,9 +331,11 @@ impl ValidatorNode for GrpcClient {
                     retry_count = 0;
                     return future::Either::Left(future::ready(true));
                 };
+                let _enter_span = span.enter();
                 if !Self::is_retryable(status) || retry_count >= max_retries {
                     return future::Either::Left(future::ready(false));
                 }
+                drop(_enter_span);
```

Comment:
Would something like this would've also worked? 🤔
```suggestion
                if !Self::is_retryable(status).instrumented(span.clone()).inner() || retry_count >= max_retries {
                    return future::Either::Left(future::ready(false));
                }
```

Reply:
This is my attempt at that https://github.com/linera-io/linera-protocol/pull/3329